bedrock/landing/templates/landing/base-landing.html

```diff
@@ -0,0 +1,43 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "base-protocol.html" %}
+
+{% block page_css %}
+  {{ css_bundle('landing-shared')}}
+{% endblock %}
+
+
+
+{% block site_header %}
+  <div class="c-navigation top-header-navigation mzp-is-sticky">
+    <div class="c-navigation-l-content">
+      <div class="c-navigation-container">
+        <div data-cta-type="navigation" data-cta-text="{{ self.page_title() }}" class="sem-landing-nav-icon c-navigation-logo mzp-c-wordmark mzp-t-wordmark-md {% block icon_class %}{% endblock %}" data></div>
```

Comment:
There's no accessible text here, it's just an empty div. And if it's not a link does it need the data attributes? There is also an extraneous `data` at the end, maybe part of an attribute that didn't get finished?

Reply:
Cleaned this up - the reason that I added the data attributes is because Dan wanted to track whether or not people were clicking on the wordmark in the nav and whether or not we'd need to add something on click in the future.